rust/src/chain_crypto/algorithms/ed25519.rs

```diff
@@ -11,7 +13,7 @@ use ed25519_bip32::XPub;
 pub struct Ed25519;
 
 #[derive(Clone)]
-pub struct Priv([u8; ed25519::SEED_LENGTH]);
+pub struct Priv([u8; ed25519::PRIVATE_KEY_LENGTH]);
```

Comment:
Just want to make sure. I understand this change will make all instance of this `Priv` to be 64 bytes instead of 32, but at the same time I am not sure if this specific `Ed25519` is used for anything at all, directly because I can only see `Ed25519Bip32` being used for all the function that are in actual use.

@rooooooooob , can you plz clarify what the `Ed25519` is being used for , compared to `Ed25519Bip32`, just so that it's extra clear about the security of this change?

Reply:
> PRIVATE_KEY_LENGTH is 64 bytes in cryptoxide 0.3, but 32 bytes in cryptoxide 0.4

![image](https://user-images.githubusercontent.com/5585355/152670925-ebd8d560-57de-41a6-8bf0-9cc50cd7ba1b.png)

Understood! This is clear now, thank you, @vincenthz !